manifests/t/ThinkscapeLimited/ZeeDrive/48.1.0.0/ThinkscapeLimited.ZeeDrive.installer.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: ThinkscapeLimited.ZeeDrive
+PackageVersion: 48.1.0.0
+Installers:
+- Architecture: x86
+  InstallerType: exe

```

```suggestion
  InstallerType: exe
  InstallerSwitches:
    Silent: Command=Install
    SilentWithProgress: Command=Install
    Upgrade: Command=Upgrade
  InstallerSuccessCodes:
  - 10
  ExpectedReturnCodes:
  - InstallerReturnCode: 11
    ReturnResponse: rebootRequiredToFinish
```

Please note, however, that this application doesn't put any entries into the ARP table so there is no way for winget to detect it once it is installed. Additionally, the only way to launch or configure the application after a silent install is to use the exe file with command line parameters. Therefore, this package probably isn't a good candidate to add to the repo at all.